parser.go

```diff
@@ -152,10 +164,20 @@ func parserFor(domain string) IParser {
 	return &DefaultParser
 }
 
-func fillIfFound(field *string, re *regexp.Regexp, text *string) {
+func fillIfFound(field *string, re *regexp.Regexp, text *string, skipWordList []string) {
 	if re != nil {
 		if val, found := findAndJoinStrings(text, re); found {
-			*field = val
+			ok := true
+
+			for _, word := range skipWordList {
+				if word == val {
```

Comment:
@pokeroklife Here we have two issues.

1. There's no break in case if skip word is equal to val
2. The variable could be defined more short: `skip = skipWord == val`